src/bulk/common.ts

```diff
@@ -550,8 +530,8 @@ function executeCommands(
     }
 
     // Merge the results together
-    const writeResult = new BulkWriteResult(bulkOperation.s.bulkResult);
     const mergeResult = mergeBatchResults(batch, bulkOperation.s.bulkResult, err, result);
+    const writeResult = new BulkWriteResult(bulkOperation.s.bulkResult);
     if (mergeResult != null) {
       return callback(undefined, writeResult);
     }
```

Comment:
```suggestion
```

optional cleanup - mergeBatchResults never returns anything, so this is dead code.

Reply:
Removed.